docs/ccoctl.md

```diff
@@ -161,3 +161,57 @@ $ ccoctl gcp delete --name=<name> --project=<gcp-project-id>
 ```
 
 where `name` is the name prefix used to create cloud resources. `project` is the ID of the gcp project.
+
+## IBMCloud
+
+### Global flags
+
+By default, the tool will output to the directory the command(s) were run in. To specify a directory, use the `--output-dir` flag.
+
+### Extract the Credentials Request objects from the above release image
+
+`ccoctl ibmcloud` can process two kind of credentials requests - `IBMCloudProviderSpec`, `IBMCloudPowerVSProviderSpec` and here are the steps to extract them from the release image
```

Comment:
Doesn't necessarily need to be in this PR or even in this doc, but for user docs I will need to know what these do/what's the difference. Looks like different platforms? I'm not 100% up-to-speed on IBM platforms.

Reply:
Yep. IBMCloudProviderSpec is for housing the cloud permissions for the IBMCloud Intel-based stuff, and IBMCloudPowerVSProviderSpec is for housing the permissions required for running on IBM Power-based stuff.

Reply:
Thanks for answering the question on my behalf, here are the enhancement proposals for these 2 platforms:

IBMCloudProviderSpec - https://github.com/openshift/enhancements/pull/773
IBMCloudPowerVSProviderSpec - https://github.com/openshift/enhancements/pull/736